src/App/App.tsx

```diff
@@ -21,20 +21,23 @@ const App: React.FC = () => {
     }
   }, [parkCode])
 
-  const generateRandomParkCode = () => {
+  const generateRandomParkCode = (): void => {
     let index = Math.floor(Math.random() * nationalParks.length)
     setParkCode(nationalParks[index].parkCode)
   }
 
   return (
     <main className="body">
-      <MainNav choosePark={choosePark}/>
+      <MainNav 
+        choosePark={choosePark}
+        generateRandomParkCode={generateRandomParkCode}/>
```

Comment:
We'll want to set router on to dynamically update on this eventually but not for this branch.